XenAdmin/Diagnostics/Checks/PoolHasDeprecatedSrsCheck.cs

```diff
@@ -55,19 +58,25 @@ public PoolHasDeprecatedSrsCheck(Host host, Dictionary<string, string> installMe
 
         protected override Problem RunHostCheck()
         {
+            if (!_manualUpgrade)
+            {
+                var hotfix = HotfixFactory.Hotfix(Host);

```

Comment:
This is needed for checks using the right version upgrade plugin as without it it may not be possible to find out the target version.

Reply:
Hmm, actually, now that I'm thinking about it, I should probably move it to the parent class because there are more checksthat may be suffering from it.